BeagleBone_black/src/communications/receiver.cpp

```diff
@@ -38,7 +38,7 @@ void ReceiverThread::run()
 
     switch (command) {
       case 1:
-        cmn_data.stopCommand = true;
+        cmn_data.module_status = data::ModuleStatus::kCriticalFailure;
```

Comment:
should not kCriticalFailure indicate loss of telemetry?

Reply:
loss of telemetry and a stop command both cause command =1, kCritical. In both cases you would want the pod to go into braking